objstore.go

```diff
@@ -455,6 +455,11 @@ func WrapWithMetrics(b Bucket, reg prometheus.Registerer, name string) *metricBu
 		bkt.ops.WithLabelValues(op)
 		bkt.opsFailures.WithLabelValues(op)
 		bkt.opsDuration.WithLabelValues(op)
+	}
+	for _, op := range []string{
+		OpGet,
+		OpGetRange,
+	} {
```

Comment:
Should we simply put this into the other loop below with transferred bytes or do we need the extra loop?